components/style/properties/longhand/box.mako.rs

```diff
@@ -351,57 +351,27 @@ ${helpers.single_keyword("overflow-clip-box", "padding-box content-box",
     spec="Internal, not web-exposed, \
           may be standardized in the future (https://developer.mozilla.org/en-US/docs/Web/CSS/overflow-clip-box)")}
 
+<%
+    overflow_custom_consts = { "-moz-scrollbars-horizontal": "SCROLLBARS_HORIZONTAL",
+                               "-moz-scrollbars-vertical": "SCROLLBARS_VERTICAL",
+                               "-moz-hidden-unscrollable": "CLIP" }
+    overflow_extra_gecko_values = """-moz-scrollbars-horizontal -moz-scrollbars-vertical
```

Comment:
What is preventing `overflow-x: -moz-scrollbars-horizontal` to be parsed?

Seems like the code handling `overflow: -moz-scrollbars-horizontal` should be in the overflow longhand instead, as @heycam said.